invenio_app_rdm/static_pages/static_pages.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio App RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Static pages."""
+import os
+from pathlib import Path
+
+from flask_webpackext import current_webpack
+from invenio_db import db
+from invenio_pages import Page
+
+
+class StaticPages:
```

Comment:
minor: we could maybe reuse the [`FixtureMixin`](https://github.com/inveniosoftware/invenio-rdm-records/blob/fbc59a9ed17b6086930cadce077c07cb752f7481/invenio_rdm_records/fixtures/fixture.py#L13).

Probably you'll need to override the `__init__` to pass the `force` flag, but otherwise, it could work out of the box by just defining the `create` method.

See also https://github.com/inveniosoftware/invenio-rdm-records/pull/1113, which loads communities, so that we can possibly align fixture loading in general